includes/notification-hubs-enable-apple-push-notifications.md

```diff
@@ -20,6 +20,10 @@ Generate the Certificate Signing Request (CSR) file, which Apple uses to generat
 1. Select **Keychain Access**, expand **Certificate Assistant**, and then select **Request a Certificate from a Certificate Authority**.
 
     ![Use Keychain Access to request a new certificate](./media/notification-hubs-enable-apple-push-notifications/notification-hubs-request-cert-from-ca.png)
+    
+> [!NOTE]
+> OS X does default to selecting the first item in whatever category you have selected within Keychain Assistant and could be problematic if you’re on the Certificates section and the Apple Worldwide Developer Relations Certification Authority is not the first item in the list. Make sure you’re in the **"Certificates"** section and select **"Apple Worldwide Developer Relations Certification Authority"** before requesting a certificate.
```

Comment:
@Grace-MacJones-MSFT Thanks -- looks good except can you please remove the word "does" in the first sentence, and just say "OS X defaults to..." Thanks for the contribution!

Reply:
Consider the following minor revision:

In Keychain Access, the first item in the list is selected by default. This could be problematic if you’re on the **Certificates** category and the **Apple Worldwide Developer Relations Certification Authority** is not the first item in the list. Make sure you have a non-key item or the **"Apple Worldwide Developer Relations Certification Authority"** key selected before generating the CSR (Certificate Signing Request).